### PR TITLE
ci: make Renovate handling tasklist.Dockerfile and operate.Dockerfile

### PR DESCRIPTION
**Pull request by houssain-barouni**
## Description

I have noticed that Renovate did not update the alpine version in tasklist.Dockerfile and operate.Dockerfile

## Related issues

closes #

### CODE REVIEW

**Review by cmur2 — APPROVED** [review]
After merge we should check if Renovate indeed correctly recognizes the new files (while still having the old as well).

**Comment by houssain-barouni** [comment]
> After merge we should check if Renovate indeed correctly recognizes the new files (while still having the old as well).

Do you have any kind of logs that shows what Renovate filters are matching?

File: .github/renovate.json [hunk 1]
```diff
@@ -157,6 +157,7 @@
     }
   ],
   "dockerfile": {
+    "fileMatch": ["(^|/)([\\w-]+\\.)?[Dd]ockerfile$"],
```
Comment (cmur2) [inline .github/renovate.json:160]: You may need to escape the `/` at position 3 in the string with `\\/`

**Comment by cmur2** [comment]
> Do you have any kind of logs that shows what Renovate filters are matching?

Hi, yes you can find two possibilities in the "Related resources" on https://github.com/camunda/camunda/wiki/CI-&-Automation#renovate

* https://github.com/camunda/camunda/issues/12605 shows an abbreviated (incomplete) list, might be on there, might be not, if not go to:
* https://developer.mend.io/github/camunda/camunda

**Comment by cmur2** [comment]
Seems working:

image

**Comment by houssain-barouni** [comment]
> Seems working:
> 
> image

but it did not create the pull requests yet, maybe we should wait a bit

**Pull request merged**